posthog/hogql/transforms/test/__snapshots__/test_in_cohort.ambr

```diff
@@ -15,11 +15,13 @@
   -- HogQL
   
   SELECT event 
-  FROM events LEFT JOIN (
-  SELECT person_id AS cohort_person_id, 1 AS matched, cohort_id 
+  FROM events 
+  WHERE and(in(person_id, (
+  SELECT person_id 
   FROM raw_cohort_people 
-  WHERE and(equals(cohort_id, XX), equals(version, 0))) AS __in_cohort ON equals(__in_cohort.cohort_person_id, person_id) 
-  WHERE and(and(1, equals(event, 'RANDOM_TEST_ID::UUID')), equals(__in_cohort.matched, 1)) 
+  WHERE equals(cohort_id, XX) 
+  GROUP BY person_id, cohort_id, version 
+  HAVING greater(sum(sign), 0))), equals(event, 'RANDOM_TEST_ID::UUID')) 
   LIMIT 100
```

Comment:
Hmm, should these snapshots have changed? The query looks different which I imagine means some query modifier has changed somewhere? Expected or nah?

Reply:
The approach in this PR doesn't work, sorry @Gilbert09  😬